openssh/map.jinja

```diff
@@ -5,27 +5,43 @@
         'config': '/etc/ssh/sshd_config',
         'lib_dir': '/usr/lib/openssh',
         'user': {},
+        'bind': { 
+          'address': '0.0.0.0',
```

Comment:
I think this desires and update in README, at least use this as default value an notify, if you don't want to bind to all interfaces, use override.

Possibly nothing wrong having this as default value in map.jinja